src/redisearch_api.c

```diff
@@ -952,6 +951,14 @@ TotalSpecsInfo RediSearch_TotalInfo(void) {
       info.gc_stats.totalCycles += gcStats.numCycles;
       info.gc_stats.totalTime += gcStats.totalMSRun;
     }
+
+    // Index errors metrics
+    size_t index_error_count = IndexSpec_GetIndexErrorCount(sp);
+    info.indexing_errors.indexing_failures += index_error_count;
+    if (info.indexing_errors.max_indexing_failures < index_error_count) {
+      info.indexing_errors.max_indexing_failures = index_error_count;
+    }
```

Comment:
Shouldn't we initialize `max_indexing_failures` before the loop?

Reply:
initialized to 0

Reply:
maybe you're right because indexing_errors is a nested struct